angular_token_auth.js

```diff
@@ -297,13 +301,19 @@
             }
         };
 
-        //use cookies if available, otherwise try localstorage
-        if (storageMethods['cookie'].test() === true) {
-            return storageMethods['cookie'];
-        } else if (storageMethods['localStorage'].test()) {
-            return storageMethods['localStorage'];
+        if (storageMethods[MODULE_SETTINGS.STORAGE_METHOD]) {
+            return storageMethods[MODULE_SETTINGS.STORAGE_METHOD];
```

Comment:
Oh wait, ack, sorry! I didn't notice there was a method being called in the conditional: I had read it as just checking it the storage method exists. I guess you need to revert